liquidation-monitor/src/agent.spec.js

```diff
@@ -922,7 +1139,11 @@ describe('process newBorrower', () => {
 
     setPriceMocks(mockBtcPrice, mockBtcCollateralFactor, mockBtcCTokenRate);
     setPriceMocks(mockEthPrice, mockEthCollateralFactor, mockEthCTokenRate);
-    await (provideHandleBlock(initializeData))();
+
+    // Set the initial alert to 0 seconds and the first block time to 15 seconds
+    initializeData.nextAlertTime = 0;
```

Comment:
```suggestion
    // Set the initial alert to 0 seconds
```

Reply:
Addressed in commit eb58f1732afc2ab71e40882b9760a78a50a8a44a